src/main/resources/schema/provider.definition.schema.v1.json

```diff
@@ -114,10 +114,43 @@
             "$ref": "base.definition.schema.v1.json#/definitions/httpsUrl"
         },
         "taggable": {
-            "description": "A boolean flag indicating whether this resource type supports tagging.",
+            "description": "(deprecated) A boolean flag indicating whether this resource type supports tagging.",
             "type": "boolean",
             "default": true
         },
+        "tagging": {
+            "type": "object",
+            "properties": {
+                "taggable": {
+                    "description": "A boolean flag indicating whether this resource type supports tagging.",
+                    "type": "boolean",
+                    "default": true
+                },
+                "tagOnCreate": {
+                    "description": "A boolean flag indicating whether this resource type supports tagging resources upon creation.",
+                    "type": "boolean",
+                    "default": true
+                },
+                "tagUpdatable": {
+                    "description": "A boolean flag indicating whether this resource type supports updatable tagging.",
+                    "type": "boolean",
+                    "default": true
+                },
+                "cloudFormationSystemTags": {
+                    "description": "A boolean flag indicating whether this resource type supports CloudFormation system tags.",
+                    "type": "boolean",
+                    "default": true
+                },
+                "tagProperty": {
+                    "description": "A reference to the Tags property in the schema.",
+                    "$ref": "http://json-schema.org/draft-07/schema#/properties/$ref"
```

Comment:
should the default value is `/properties/Tags`? we can validate it as well to ensure there is `/properties/Tags` otherwise, they have to configure the right  configure of it.

Reply:
sure already changed this and added validation